### ISSUE TITLE
Keybase on Ubuntu 16.04 is broken

### ISSUE THREAD

**Issue by jacnoc** [keybase/client#6367]
Tried to remove a device (my Windows 10 machine) using my Ubuntu server and got this.

>  ERROR dial unix /run/user/1000/keybase/keybased.sock: connect: no such file or directory

**Comment by maxtaco** [comment]
Did you run `run_keybase.sh`?

Cc: @oconnor663 and @cjb

**Comment by oconnor663** [comment]
`keybase log send` would be very helpful if you get a chance. I suspect something along the lines of "`XDG_RUNTIME_DIR` doesn't actually exist".

**Comment by cjb** [comment]
Yeah, I'd guess that you forgot to run `run_keybase` on the Ubuntu server.
